packages/css/lib/index.ts

```diff
@@ -1,14 +1,14 @@
-import binding = require("./binding");
+import * as binding from "./binding";
 
 export async function minify(
   content: Buffer,
   options: any
 ): Promise<binding.TransformOutput> {
-  return binding.minify(content, toBuffer(options));
+  return binding.minify(content, toBuffer(options ?? {}));
 }
 
 export function minifySync(content: Buffer, options: any) {
-  return binding.minifySync(content, toBuffer(options));
+  return binding.minifySync(content, toBuffer(options ?? {}));
```

Comment:
Allow to avoid options for better DX (the same for other)

Reply:
Would we be better off using `Option<String>` in Rust?